core/translations/en.json

```diff
@@ -372,6 +374,7 @@
     "ethereum__title_confirm_message": "Confirm message",
     "ethereum__title_confirm_struct": "Confirm struct",
     "ethereum__title_confirm_typed_data": "Confirm typed data",
+    "ethereum__title_confirm_typed_hashes": "Confirm typed hashes",
```

Comment:
could alternatively remove this and just reuse `ethereum__title_confirm_typed_data`